parking/shared/rest_models.py

```diff
@@ -12,17 +20,47 @@ class ParkingLot:
     id: int = attr.ib(validator=attr.validators.instance_of(int), default=0)
 
 
+@attr.s
+class RESTError:
```

Comment:
Like I said earlier it's probably easier to just have tornado give back a 400 code with an error message than to try define a set of responses for each possible way of making a malformed request?

Or if we did, they probably don't need to be separate classes anyway since they contain no instance data. (InvalidLotIDError could be an instance of RESTError for example ...)